proposals/2021-02-enable-ibc-transfers/proposal.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Enable IBC transfers",
+  "description": "The Proposal enables transfering and recieving assets using the IC20 protocol on the Cosmos Hub. If this proposal passes, there will be IBC assets available in the Bank module of the Hub and ATOM will be available on Zones connected over IBC.\n Iqlusion believes that the IBC software is suffciently stable for small amounts of value transfer. We expect there to be issues with stuck funds and UX confusion but overcoming these issues will only happen once IBC is live.\n Security Model\n Tendermint full nodes produce agreement under the assumption that at most ⅓ of the voting power held by validators is Byzantine.\n IBC\nIBC is a protocol for authenticated message passing between heterogeneous sovereign blockchains. IBC requires trusting that chains on both sides of the connections operate within their security model.\n Incentive Security Extensions\n IBC has a facility to support freezing connections once a violation of the security model has occurred. The set of criteria for detecting such attacks continues to evolve and is a constant focus of research.",
+  "changes": [
+    {
+      "subspace": "transfer",
+      "key":"send_enabled",
```

Comment:
Not sure if this should match with the keys below

```go
var (
	// KeySendEnabled is store's key for SendEnabled Params
	KeySendEnabled = []byte("SendEnabled")
	// KeyReceiveEnabled is store's key for ReceiveEnabled Params
	KeyReceiveEnabled = []byte("ReceiveEnabled")
)
```

Reply:
presumably it should unless there is some kind of conversion

Reply:
It needs to match, this is the proposal format I have tested:
```
  "changes": [
    {
      "subspace": "transfer",
      "key": "SendEnabled",
      "value": true
    }
  ],
```

I will retest in the next week

Reply:
Tested, the original key will result in:
```
{"height":"20359","txhash":"FEF5ED577586B1542144C3C13927F4CCBB9AB793176F11BD4C08375276FFF48B","codespace":"undefined","code":111222,"data":"","raw_log":"panic message redacted to hide potentially sensitive system info: panic","logs":[],"info":"","gas_wanted":"200000","gas_used":"50231","tx":null,"timestamp":""}
```

Using `SendEnabled` and `ReceiveEnabled` worked fine

Reply:
Yeah, those are the right strings.

Reply:
Fixed